web/gui/dashboard_info.js

```diff
@@ -3803,10 +3803,30 @@ netdataDashboard.context = {
     // POSTGRESQL
     'postgres.connections_utilization': {
-        info: 'Connections in use as percentage of <i>max_connections</i>. Connection "slots" that are reserved for superusers (<i>superuser_reserved_connections</i>) are subtracted from the limit. If the utilization is 100% new connections will be accepted only for superusers, and no new replication connections will be accepted.'
+        room: { 
+            mainheads: [
+                function (_, id) {
+                    cgroupMemLimitIsSet = 1;
+                    return '<div data-netdata="' + id + '"'
+                        + ' data-append-options="percentage"'
+                        + ' data-gauge-max-value="100"'
+                        + ' data-chart-library="gauge"'
+                        + ' data-title="Connections Utilization"'
+                        + ' data-units="%"'
+                        + ' data-gauge-adjust="width"'
+                        + ' data-width="12%"'
+                        + ' data-before="0"'
+                        + ' data-after="-CHART_DURATION"'
+                        + ' data-points="CHART_DURATION"'
+                        + ' data-colors="' + NETDATA.colors[1] + '"'
+                        + ' role="application"></div>';
+                }
+            ],
+        },
+        info: '<p>A connection is an established line of communication between a client and the PostgreSQL server. Each connection adds to the load on the PostgreSQL server. To guard against running out of memory or overloading the database the <i>max_connections</i> parameter (default = 100) defines the maximum number of concurrent connections to the database server. A separate parameter, <i>superuser_reserved_connections</i> (default = 3), defines the quota for superuser connections (so that superusers can connect even if all other connection slots are blocked).</p><p><br></p><p><b>Total connection utilization</b> across all databases. Utilization is measured as a percentage of (<i>max_connections</i> - <i>superuser_reserved_connections</i>). If the utilization is 100% no more new connections will be accepted (superuser connections will still be accepted if superuser quota is available).</p>'
     },
     'postgres.connections_usage': {
-        info: '<p>Connections usage. The maximum number of concurrent connections to the database server is <i>max_connections</i> minus <i>superuser_reserved_connections</i>.</p><p><b>Available</b> - new connections allowed. <b>Used</b> - connections currently in use.</p>'
+        info: '<p><b>Connections usage</b> across all databases. The maximum number of concurrent connections to the database server is (<i>max_connections</i> - <i>superuser_reserved_connections</i>). As a general rule, if you need more than 200 connections it is advisable to use connection pooling.</p><p><b>Available</b> - new connections allowed. </p><p><b>Used</b> - connections currently in use.</p>'
```

Comment:
Are you sure about a paragraph per dimension? We didn't do it because it doesn't really look good when there are 4+ dimensions (with different description lengths) and consumes a lot of space. And I wouldn't say it improves readability, I think it is pretty easy (at least for me) to find the needed dimension name because we made them in bold font.


<img width="1306" alt="Screenshot 2022-08-18 at 17 26 15" src="https://user-images.githubusercontent.com/22274335/185419679-50291e75-ee30-4f65-8d59-3390a0c0f9df.png">

Reply:
Hmm, you are right about charts with 4+ dimensions. But for the ones with <=2 dimensions do we really need to club them all together in the same sentence? 

Or you think consistency is more important?

Reply:
For me personally, it is 
- consistency
- a paragraph per dimension (we did try it with @vlvkobal and came to the conclusion that it is much better in theory than in reality) does not improve readability but the opposite - the text stands out more, which is annoying when you look at the chart not for the first time.
- consumes less space, and the text cannot be removed.

Reply:
Alternatively - should descriptions about dimensions just live as a tooltip when user hovers over the dimension name in chart legend. @ilyam8

Reply:
OK I've removed the <p></p> so they are back to same line now. @ilyam8